### PR TITLE
Added io polyfill and Safari-only cape style.

### PR DESCRIPTION
**Pull request by peytonchance**
This PR adds the IntersectionObserver polyfill and implements the only workaround for Safari-specific styles that I could get working - the cape is shifted down and right in Safari without the fix.

image

### CODE REVIEW

**Review by olets — COMMENTED** [review]
I wonder if it's something about the SVG's bounding boxes at the various points in the animation? Is it shifted in Safari without the animation? Is it shifted in Safari without the animation, if you use the non-default path?

Since you've added intersection observer as a dependency, is it possible to `require` it rather than including that js file? (I haven't used IO and didn't look carefully at the added file, maybe this is just how it has to work)

**Comment by peytonchance** [comment]
@olets  - I just checked and it is shifted in Safari without the animation while using the non-default path.

**Comment by olets** [comment]
Sounds promising. You might try adding extra space around the Florence image so that the path change doesn't change the bounding box. I'd ask someone on Design for suggestions about how to do that

Tangent: happened to notice that https://github.com/peytonchance/florence-build/blob/master/src/images/florence.svg already has trimmable whitespace. Haven't looked that the other graphics, but they might too. An easy way trim that is drop an SVG into Figma, select it, and export it as SVG, and then run through svgo/svgomg. Don't know how badly that'll affect your css/js… might not be worth it

File: src/html/layouts/application.html [hunk 1]
```diff
@@ -13,7 +13,17 @@
     {% block content %}{% endblock %}
 
     {% block javascript %}
+      <script src="https://polyfill.io/v2/polyfill.min.js?features=IntersectionObserver"></script>
       <script src="javascripts/app.js"></script>
     {% endblock %}
+    <style>
+        @media not all and (min-resolution:.001dpcm) {
```
Comment (jeremyfrank) [inline src/html/layouts/application.html]: A good practice is to add a short comment about what this block does.

Comment (jeremyfrank) [reply src/html/layouts/application.html]: Or why it's necessary.

File: src/html/layouts/application.html [hunk 2]
```diff
@@ -13,7 +13,17 @@
     {% block content %}{% endblock %}
 
     {% block javascript %}
+      <script src="https://polyfill.io/v2/polyfill.min.js?features=IntersectionObserver"></script>
```
Comment (jeremyfrank) [inline src/html/layouts/application.html]: Seems like you have 3 different polyfills going on here:
1. the polyfill from polyfill.io
2. the polyfill specified in package.json (but not imported anywhere)
3. the polyfill in src/javascripts/intersection-observer/intersection-observer.js

Unless I'm missing something, you should only need one of these.

**Comment by jeremyfrank** [comment]
Oops, looks like you already took care of the duplicate polyfill stuff. Carry on! 👍

**Comment by jeremyfrank** [comment]
+1 to Henry's suggestion on making the SVG artboard (viewbox) larger in Illustrator, with enough room for the cape to move. That may take care of the cape shifting issue in Safari.

**Comment by peytonchance** [comment]
Yup it still works after SVGO - I went back to look at the path values and made sure the starting and original matched, and then the first value of the target path was M105 when it should have been ~M130. I changed them manually and it seemed to do the trick.

**Pull request merged**